Presentation.Web/app/app.ts

```diff
@@ -30,7 +30,8 @@ app.config([
     "$httpProvider",
     "$windowProvider",
     "notifyProvider",
-    ($httpProvider, $windowProvider, notifyProvider) => {
+    "$locationProvider",
+    ($httpProvider, $windowProvider, notifyProvider, $locationProvider) => {
```

Comment:
Hvorfor er der ikke "$" foran notifyProvider?

Reply:
.. https://stackoverflow.com/questions/12648543/angularjs-and-its-use-of-dollar-variables